sdk/cosmos/azure-cosmos/samples/examples.py

```diff
@@ -0,0 +1,145 @@
+# These examples are ingested by the documentation system, and are
+# displayed in the SDK reference documentation. When editing these
+# example snippets, take into consideration how this might affect
+# the readability and usability of the reference documentation.
+
+# All interaction with Cosmos DB starts with an instance of the CosmosClient
+# [START create_client]
+from azure.cosmos import errors, CosmosClient, PartitionKey
+
+import os
+
+url = os.environ["ACCOUNT_URI"]
+key = os.environ["ACCOUNT_KEY"]
+client = CosmosClient(url, key)
+# [END create_client]
+
+# Create a database in the account using the CosmosClient,
+# specifying that the operation shouldn't throw an exception
+# if a database with the given ID already exists.
+# [START create_database]
+database_name = "testDatabase"
+try:
+    database = client.create_database(id=database_name)
+except errors.CosmosResourceExistsError:
+    database = client.get_database_client(database=database_name)
+# [END create_database]
+
+# Create a container, handling the exception if a container with the
+# same ID (name) already exists in the database.
+# [START create_container]
+container_name = "products"
+try:
+    container = database.create_container(
+        id=container_name, partition_key=PartitionKey(path="/productName")
+    )
+except errors.CosmosResourceExistsError:
+    container = database.get_container_client(container_name)
+# [END create_container]
+
+# Create a container with custom settings. This example
+# creates a container with a custom partition key.
+# [START create_container_with_settings]
+customer_container_name = "customers"
+try:
+    customer_container = database.create_container(
+        id=customer_container_name,
+        partition_key=PartitionKey(path="/city"),
+        default_ttl=200,
+    )
+except errors.CosmosResourceExistsError:
+    customer_container = database.get_container_client(customer_container_name)
+# [END create_container_with_settings]
+
+# Retrieve a container by walking down the resource hierarchy
+# (client->database->container), handling the exception generated
+# if no container with the specified ID was found in the database.
+# [START get_container]
+database = client.get_database_client(database_name)
+container = database.get_container_client(container_name)
+# [END get_container]
+
+# [START list_containers]
+database = client.get_database_client(database_name)
+for container in database.list_containers():
+    print(f"Container ID: {container.id}")
```

Comment:
f-string, is this expected to work with python < 3.6?

Reply:
Looks like three more lines below have as well

Reply:
also need future `print_function` if this is to work also on 2.7

Reply:
Cool - I will use old-school formatting